server/routes/probationPractitionerRoutes.ts

```diff
@@ -22,7 +22,11 @@ export default function probationPractitionerRoutes(router: Router, services: Se
     probationPractitionerReferralsController.showInterventionProgress(req, res)
   )
   get(router, '/referrals/:id/details', (req, res) => probationPractitionerReferralsController.showReferral(req, res))
+  // Legacy route to keep links in old emails still working. We'll monitor and remove once traffic drops off
   get(router, '/action-plan/:actionPlanId/appointment/:sessionNumber/post-session-feedback', (req, res) =>
+    probationPractitionerReferralsController.viewLegacySubmittedPostSessionFeedback(req, res)
+  )
+  get(router, '/referrals/:referralId/appointment/:sessionNumber/post-session-feedback', (req, res) =>
```

Comment:
I'm just wondering if there is plan for an equivalent change for SP as well (in a different ticket)?

I realise it's less important due to emails, but I do like how appointments are connected to a referral than an action plan.

Reply:
I think this was influenced by a conversation between @rgforsyth and @tomsmyers - there was a corresponding back-end change for the SP side too?

Reply:
Sorry, just seen this. The backend change for this is in the Step 3 backend pr that we plan to merge soon(pending review) https://github.com/ministryofjustice/hmpps-interventions-service/pull/578

We made this change as it was the only pp url used in the backend that wasn't referenced by referral. We wanted to make it consistent. I think the sp url's are in the same form?